<ui version="4.0" >
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Git Ouput</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="output" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>OutputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel" >
     <x>245</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
